clients/pkg/promtail/scrapeconfig/scrapeconfig.go

```diff
@@ -202,15 +211,19 @@ type SyslogTargetConfig struct {
 	// message should be pushed to Loki
 	UseRFC5424Message bool `yaml:"use_rfc5424_message"`
```

Comment:
I think we wanted a second new config option `PushFullSyslogMessage`? we still need to keep this one as well, and mark it as deprecated

Reply:
I can't do it, because RFC3164 message has lack of `String()` method.  See implementation for RFC5424: https://github.com/leodido/go-syslog/blob/v4.1.0/rfc5424/builder.go#L9348-L9408

The issue that RFC3164 isn't well structured and quite flexible, and rebuild original message is quite a challenge.

Sorry to misslead you.